specs/credentialmanagement/index.src.html

```diff
@@ -238,7 +238,7 @@ <h4 id="examples-password-signin">Password-based Sign-in</h4>
     <div class="example">
       <pre>
         navigator.<a attribute>credentials</a>.<a idl lt="get()">get</a>({
-          "<a idl>types</a>": [ "password" ]
+          "<a idl>filters</a>": { "password" : {} }
```

Comment:
What do you suggest here? I'm looking for something that says "don't apply any filters".
How about

```javascript
navigator.credentials.get({
  filters : { "password" : "return all" }
```

Ultimately, it's not that important as the `PasswordCredential` is a special case as it isn't filtered. You either want them or you don't, you can't filter the set further.